test/support/api/program-flows.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'underscore';
+import { getResource, getRelationship } from 'helpers/json-api';
+
+Cypress.Commands.add('routeProgramFlows', (mutator = _.identity, programId) => {
+  cy
+    .fixture('collections/program-flows').as('fxProgramFlows')
```

Comment:
don't know if you've caught this yet, but you'll need to add a generator for this collection here: https://github.com/RoundingWell/care-ops-frontend/tree/develop/test/fixtures/config